src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25859.cs

```diff
@@ -0,0 +1,25 @@
+﻿#if !MACCATALYST
```

Comment:
Why avoid Catalyst?, is crashing?

For now, this test will not do much on macOS because the `VerifyScreenshot` is not yet implemented.

Reply:
Since `VerifyScreenshot` was not implemented on Mac, I had previously restricted it. I have now removed that restriction